packages/ui/components/data-table/DataTableSelectionBar.tsx

```diff
@@ -0,0 +1,31 @@
+import type { Table } from "@tanstack/react-table";
+
+import type { SVGComponent } from "@calcom/types/SVGComponent";
+
+import { Button } from "../button";
+
+interface DataTableSelectionBarProps<TData> {
+  table: Table<TData>;
+  actions?: {
+    label: string;
+    onClick: () => void;
+    icon?: SVGComponent;
+  }[];
+}
+
+export function DataTableSelectionBar<TData>({ table, actions }: DataTableSelectionBarProps<TData>) {
```

Comment:
Bottom selection bar used for bulk actions see figma: https://www.figma.com/file/ZJNAFHCAq6rnuGRgUFxjzQ/Organizations----New-features?type=design&node-id=4709-210337&mode=design&t=TCtyQnThwj4nteAr-4